wcfsetup/install/files/lib/system/database/table/column/CharDatabaseTableColumn.class.php

```diff
@@ -0,0 +1,34 @@
+<?php
+namespace wcf\system\database\table\column;
+
+/**
+ * Represents a `char` database table column.
+ * 
+ * @author	Matthias Schmidt
+ * @copyright	2001-2019 WoltLab GmbH
+ * @license	GNU Lesser General Public License <http://opensource.org/licenses/lgpl-license.php>
+ * @package	WoltLabSuite\Core\System\Database\Table\Column
+ * @since	5.2
+ */
+class CharDatabaseTableColumn extends AbstractDatabaseTableColumn implements ILengthDatabaseTableColumn {
+	use TLengthDatabaseTableColumn;
+	
+	/**
+	 * @inheritDoc
+	 */
+	protected $type = 'char';
+	
+	/**
+	 * @inheritDoc
+	 */
+	public function getMaximumLength() {
+		return 255;
+	}
+	
+	/**
+	 * @inheritDoc
+	 */
+	public function getMinimumLength() {
+		return 0;
```

Comment:
Does `0` even make any sense for this type? Regardless of wether or not MySQL permits this, it makes no sense and thus implies a programming error. This applies to other column types with a length attribute (again, regardless if MySQL actually cares about it!).